stix_shifter_modules/darktrace/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,1762 @@
+		],
+		"conn_state": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.conn_state",
+				"object": "nt",
+				"transformer": "ConnStateToDesc"
+			}
+		],
+		"conn_state_full": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.connection_state_desc",
+				"object": "nt"
+			}
+		],
+		"local_orig": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.is_locally_originated",
+				"object": "nt"
+			}
+		],
+		"local_resp": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.is_locally_responded",
+				"object": "nt"
+			}
+		],
+		"missed_bytes_orig": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.missed_bytes_orig",
+				"object": "nt"
+			}
+		],
+		"missed_bytes_resp": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.missed_bytes_resp",
+				"object": "nt"
+			}
+		],
+		"history": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.history",
+				"object": "nt"
+			}
+		],
+		"orig_pkts": [
+			{
+				"key": "network-traffic.src_packets",
+				"object": "nt"
+			}
+		],
+		"orig_ip_bytes": [
+			{
+				"key": "network-traffic.src_byte_count",
+				"object": "nt"
+			}
+		],
+		"resp_pkts": [
+			{
+				"key": "network-traffic.dst_packets",
+				"object": "nt"
+			}
+		],
+		"resp_ip_bytes": [
+			{
+				"key": "network-traffic.dst_byte_count",
+				"object": "nt"
+			}
+		],
+		"orig_ttl": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.originator_ttl",
+				"object": "nt"
+			}
+		],
+		"resp_ttl": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.responder_ttl",
+				"object": "nt"
+			}
+		],
+		"tunnel_parents": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.tunnel_parents",
+				"object": "nt"
+			}
+		],
+		"orig_percent_invalid_checksum": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.orig_percent_invalid_checksum",
+				"object": "nt"
+			}
+		],
+		"resp_percent_invalid_checksum": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.resp_percent_invalid_checksum",
+				"object": "nt"
+			}
+		],
+		"outer_vlan": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.outer_vlan",
+				"object": "nt"
+			}
+		],
+		"vlan": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.vlan_id",
+				"object": "nt"
+			}
+		],
+		"orig_cc": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.orig_country_code",
+				"object": "nt"
+			}
+		],
+		"resp_cc": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.resp_country_code",
+				"object": "nt"
+			}
+		],
+		"orig_asn": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.originator_asn",
+				"object": "nt"
+			}
+		],
+		"resp_asn": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-conn.responder_asn",
+				"object": "nt"
+			}
+		]
+	},
+	"ssl": {
+		"version": [
+			{
+				"key": "network-traffic.extensions.x-darktrace-ssl.ssl_version",
```

Comment:
all these ssl fields shouldn't be an extension of `network-traffic` object. should be an extension of `x509-certificate`. there are bunch of extension types defined in `x509-certificate` if none of them matches then you can move them to its own custom extension.